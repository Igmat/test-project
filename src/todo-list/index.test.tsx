import React from 'react';
import renderer from 'react-test-renderer';
import TodoList from './';

test('TodoList matches snapshot', () => {
  const tree = renderer
    .create(<TodoList />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});