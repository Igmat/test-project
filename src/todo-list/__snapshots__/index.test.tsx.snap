// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList matches snapshot 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <h2
      className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
    >
      TODO List
    </h2>
    <ul
      className="MuiList-root MuiList-padding"
    >
      <li
        className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
        disabled={false}
      >
        <form
          onSubmit={[Function]}
        >
          <div
            className="MuiFormControl-root MuiTextField-root"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              New TODO
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="name"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value=""
              />
            </div>
          </div>
        </form>
      </li>
    </ul>
  </div>
  <div
    className="MuiCardActions-root MuiCardActions-spacing"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Clear
      </span>
    </button>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-alignRight"
    >
      Comppleted: 
      0
      /
      0
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-alignRight"
    >
      TODO: 
      0
      /
      0
    </p>
  </div>
</div>
`;
